CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** [Open an Issue]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Issue]
+- **💻 I want to code:** See below
+
+If you like the project, but just don't have time to contribute, that's OK too!
+You can also star the project ⭐, rave about it online 💬, or add a link to us
+🔗 in your project's readme.
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead, please [report them to the Microsoft Security Response Center]. You can
+read more about our security policy in [`SECURITY.md`].
+
+## Contributing code
+
+👨‍💼 This repository is a collection of devcontainer features managed by
```

Comment:
We refer to dev containers as 2 words, so it'd be fantastic if you could update this PR to refer to them as "dev containers" 😄

Reply:
Fixed in https://github.com/jcbhmr/features/commit/d7861eee339f77ce5b5762a3fe84baed642ecaca